core/appmodule/environment.go

```diff
@@ -16,4 +17,5 @@ type Environment struct {
 	HeaderService   header.Service
 	KVStoreService  store.KVStoreService
 	MemStoreService store.MemoryStoreService
+	Logger          log.Logger
```

Comment:
I would have expected changes in runtime due to this new field, or is it going to be merged after https://github.com/cosmos/cosmos-sdk/pull/19414 ?

Reply:
This was the first iteration but #19414 iterates over it again. Either I need to reflect those changes here or wait for the #19414 to be merged